src/libs/UserUtils.js

```diff
@@ -201,6 +201,15 @@ function getSmallSizeAvatar(avatarURL, accountID) {
     return `${source.substring(0, lastPeriodIndex)}_128${source.substring(lastPeriodIndex)}`;
 }
 
+/**
+ * Generate a random accountID.
+ *
+ * @returns {String}
+ */
+function generateAccountID() {
+    return Math.floor(Math.random() * 99999999).toString();
```

Comment:
We should modify this slightly to be a random number that starts at an already very high number, since we know that lower accountIDs have already been used and we don't want to dupe anything :D

Reply:
Hum, perhaps I should just use the approach of `generateReportID`

Reply:
I like that plan 👍